mediation-old/src/main/java/com/wso2telco/dep/mediator/impl/ussd/SendUSSDHandler.java

```diff
@@ -164,9 +167,10 @@ public boolean handle(MessageContext context) throws CustomException, AxisFault,
 
 	private void validateUssdAction(JSONObject jsonBody, MessageContext context) throws Exception {
 		String ussdAction = jsonBody.getJSONObject("outboundUSSDMessageRequest").getString("ussdAction");
-		if ( !(ussdAction.equals(MTINIT) || ussdAction.equals(MTCONT)) ){
+		if ( !(ussdAction.equals(MTINIT) || ussdAction.equals(MTCONT) || ussdAction.equals(MTFIN)  || ussdAction.equals(MOCONT) || ussdAction.equals(MOFIN))){
```

Comment:
IMO,
01. This part needs to be implemented in oneapi-validation module since this is a validation.
02. This handle will be hit only when sending ussd request by SP (service provider), hence there cannot be different ussd actions other than "mtinit". Hence it's better it can be handled properly than this
03. This error occurred due to issue in the SP's request, it shuld be returned "400 bad request" Http status code, nt 405 error code

Reply:
- Will move to one-api validation
- WIll check validate actions and update spec and code

Reply:
done